package-lock.json

```diff
@@ -896,9 +896,9 @@
       }
```

Comment:
We might need to account for backward compatibility in this change. Have we checked?